modules/Book.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
```

Comment:
Same here kindly remove these lines of code and fix the linter issues. :+1: :smiley: :smiley: Please do the same in the rest of the js files.
![image](https://user-images.githubusercontent.com/98363075/200565517-29f63e50-cb16-4859-8e6e-b6c96a1ff31e.png)